Description: Increase DHCP retries for Foxconn routers
 Routers using the Foxconn code base enable the ethernet switch more than 30
 seconds before enabling the DHCP server during bootup.  This causes the DHCP
 request to time out, and for an IPv4LL address to be used.  This is really a
 bug in connman, but we'll work around it for now by trying even harder to get
 a DHCP address.
 .
 connman (1.12-netgear1) unstable; urgency=low
 .
   * Update to upstream release 1.12.
   * Increase DHCP retries for Foxconn routers.
Author: Justin Maggard <justin.maggard@netgear.com>

---

--- connman-1.12.orig/gdhcp/client.c
+++ connman-1.12/gdhcp/client.c
@@ -46,7 +46,7 @@
 #include "ipv4ll.h"
 
 #define DISCOVER_TIMEOUT 5
-#define DISCOVER_RETRIES 6
+#define DISCOVER_RETRIES 12
 
 #define REQUEST_TIMEOUT 5
 #define REQUEST_RETRIES 3
